app/Controllers/Http/EventsController.ts

```diff
@@ -18,12 +18,15 @@ export default class EventsController {
   }
 
   // post one event
-  public async store({ request, response }: HttpContextContract) {
+  public async store({ request, response, auth }: HttpContextContract) {
     try {
       const payload = await request.validate(CreateEventValidator)
-      // TODO set is_public if user.role equal moderator or admin
-      // TODO create_email must not be handed over, must be taken over via the auth
       const event = await Event.create(payload)
+      if (auth.user) {
+        // TODO set is_public if user.role equal moderator or admin
```

Comment:
Todo kann ja jetzt weg :)

Reply:
das muss ich noch machen, ich habe in diesen PR noch keine user Rollen. ich würde das dann nach dem rebase machen oder in einen anderen PR

Reply:
ah ja, hab mich verguckt